src/bokeh/models/callbacks.py

```diff
@@ -130,8 +130,9 @@ def __init__(self, *args, **kwargs) -> None:
     The code is made into the body of a function, and all of of the named objects in
     ``args`` are available as parameters that the code can use. Additionally,
     a ``cb_obj`` parameter contains the object that triggered the callback
-    and an optional ``cb_data`` parameter that contains any tool-specific data
-    (i.e. mouse coordinates and hovered glyph indices for the ``HoverTool``).
+    an optional ``cb_data`` parameter that contains any tool-specific data
+    (i.e. mouse coordinates and hovered glyph indices for the ``HoverTool``)
+    an additional context in ``cb_context`` argument.
```

Comment:
```suggestion
    and additional document context in ``cb_context`` argument.
```

I think it might make sense to call this specifically "document context" throughout? That seems to fit and would be less generic than "context" which could mean almost anything.